src/main/java/org/folio/rest/impl/RTACResourceImpl.java

```diff
@@ -31,23 +31,14 @@
 import io.vertx.core.logging.Logger;
 import io.vertx.core.logging.LoggerFactory;
 
-public final class RTACResourceImpl implements RTACResource {
+public final class RTACResourceImpl implements Rtac {
```

Comment:
Is "Rtac" a new interface or a renamed one because I'm not seeing the interface Rtac in these changes? I guess it was added or changed previously? If so then how could the code have worked when it might not have been able to compile?

Reply:
The interface is generated via RMB at build time and is never checked in. It was generating it as `RTACResource` in RAML0.8, but in RAML1.0, it drops the `Resource` part of the name and I guess it doesn't preserve case.